src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UseCountProperly.cs

```diff
@@ -403,40 +412,39 @@ private static void ReportCA1836(OperationAnalysisContext context, string operat
                 bool shouldNegateAny = !shouldNegateIsEmpty;
                 ReportCA1828(context, shouldNegateAny, operationKey!, methodName, parent);
```

Comment:
In this case, we still want to report within expression tree (at least I think) because I don't expect `CountAsync` to do something specific in queryable contexts.